src/library/dao/trackdao.cpp

```diff
@@ -57,6 +58,15 @@ void markTrackLocationsAsDeleted(QSqlDatabase database, const QString& directory
     }
 }
 
+QString joinTrackIdList(const QSet<TrackId>& trackIds) {
+    QStringList trackIdList;
+    trackIdList.reserve(trackIds.size());
+    for (const auto& trackId : trackIds) {
+        trackIdList.append(trackId.toString());
+    }
+    return trackIdList.join(QChar(','));
```

Comment:
This whole function can be:

trackIds.values().join(',');

Reply:
No. This function is only provided by QStringList.